Tools/cases_generator/generate_cases.py

```diff
@@ -53,8 +53,9 @@ def write_raw(self, s: str) -> None:
         self.stream.write(s)
 
     def emit(self, arg: str) -> None:
+        prefix = '' if arg.startswith('#') else self.prefix
```

Comment:
There's actually no requirement that `#ifdef` etc. start in column 0. It might look less hideous if you just indent it with the code (and do the same in bytecodes.c).